project_stock/security/project_stock_security.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="account_analytic_line_project_stock_rule" model="ir.rule">
+        <field name="name">account.analytic.line.project_stock</field>
+        <field name="model_id" ref="analytic.model_account_analytic_line" />
+        <field name="domain_force">[('stock_task_id', '!=', False)]</field>
+        <field
+            name="groups"
+            eval="[(4, ref('analytic.group_analytic_accounting')), (4, ref('project.group_project_manager'))]"
```

Comment:
Why is this needed? This should be bypassed other way (with `sudo` for example).

Reply:
This is necessary because if `hr_timesheet` addon is installed, the analytic lines generated according to the rules cannot be accessed: https://github.com/odoo/odoo/blob/15.0/addons/hr_timesheet/security/hr_timesheet_security.xml

Reply:
And that's the idea... HR timesheet users only see timesheet entries, not these ones. Only accounting managers with full access will see all the analytic entries.

Reply:
Ok, removed